src/content/learn/editor-setup.md

```diff
@@ -1,62 +1,62 @@
+અન્ય યોગદાનકર્તા સાથે તમારો કોડ શેર કરતી વખતે તમે જે છેલ્લું કામ કરવા માંગો છો તે છે [tabs vs spaces](https://www.google.com/search?q=tabs+vs+spaces) વિશે ચર્ચામાં આવવું! સદનસીબે, [Prettier](https://prettier.io/) તમારા કોડને પ્રીસેટ, ગોઠવણી કરી શકાય તેવા નિયમોને અનુરૂપ થવા માટે તેને ફરીથી ફોર્મેટ કરીને સાફ કરશે. Prettier ચલાવો, અને તમારી બધી ટેબ્સ સ્પેસમાં રૂપાંતરિત થઈ જશે-અને તમારું ઇન્ડેન્ટેશન, અવતરણ વગેરે પણ ગોઠવણીને અનુરૂપ થવા માટે બદલાઈ જશે. આદર્શ સેટઅપમાં, જ્યારે તમે તમારી ફાઇલ સાચવશો ત્યારે Prettier ચાલશે, તમારા માટે આ સંપાદનો ઝડપથી કરશે.
 
-You can install the [Prettier extension in VSCode](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) by following these steps:
+તમે આ પગલાંને અનુસરીને [VSCode માં Prettier એક્સટેન્શન](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) ઇન્સ્ટોલ કરી શકો છો:
 
-1. Launch VS Code
-2. Use Quick Open (press Ctrl/Cmd+P)
-3. Paste in `ext install esbenp.prettier-vscode`
-4. Press Enter
+1. VS Code લોંચ કરો
+2. ક્વિક ઓપનનો ઉપયોગ કરો (Ctrl/Cmd+P દબાવો)
+3. `ext install esbenp.prettier-vscode` માં પેસ્ટ કરો
+4. Enter દબાવો
 
-#### Formatting on save {/*formatting-on-save*/}
+#### સેવ પર ફોર્મેટિંગ {/*formatting-on-save*/}
 
-Ideally, you should format your code on every save. VS Code has settings for this!
+આદર્શ રીતે, તમારે દરેક સેવ પર તમારો કોડ ફોર્મેટ કરવો જોઈએ. VS Code આ માટે સેટિંગ્સ ધરાવે છે!
 
-1. In VS Code, press `CTRL/CMD + SHIFT + P`.
-2. Type "settings"
-3. Hit Enter
-4. In the search bar, type "format on save"
-5. Be sure the "format on save" option is ticked!
+1. VS કોડમાં, `CTRL/CMD + SHIFT + P` દબાવો.
+2. "સેટિંગ્સ" લખો
+3. એન્ટર દબાવો
+4. સર્ચ બારમાં, "સેવ પર ફોર્મેટ" ટાઈપ કરો
+5. ખાતરી કરો કે "સેવ પર ફોર્મેટ" વિકલ્પ ટિક કરેલ છે!
 
-> If your ESLint preset has formatting rules, they may conflict with Prettier. We recommend disabling all formatting rules in your ESLint preset using [`eslint-config-prettier`](https://github.com/prettier/eslint-config-prettier) so that ESLint is *only* used for catching logical mistakes. If you want to enforce that files are formatted before a pull request is merged, use [`prettier --check`](https://prettier.io/docs/en/cli.html#--check) for your continuous integration.
+> જો તમારા ESLint પ્રીસેટમાં ફોર્મેટિંગ નિયમો હોય, તો તેઓ Prettier સાથે વિરોધાભાસી થઈ શકે છે. અમે [`eslint-config-prettier`](https://github.com/prettier/eslint-config-prettier) નો ઉપયોગ કરીને તમારા ESLint પ્રીસેટમાંના તમામ ફોર્મેટિંગ નિયમોને અક્ષમ કરવાની ભલામણ કરીએ છીએ જેથી ESLintનો *ફક્ત* ઉપયોગ તાર્કિક ભૂલો પકડવા માટે થાય. જો તમે પુલ વિનંતીને મર્જ કરવામાં આવે તે પહેલાં ફાઇલોને ફોર્મેટ કરવામાં આવે તે લાગુ કરવા માંગતા હો, તો તમારા સતત એકીકરણ માટે [`prettier --check`](https://prettier.io/docs/en/cli.html#--check) નો ઉપયોગ કરો. .
```

Comment:
અક્ષમ = નિષ્ક્રિય, people might not understand high level word like અક્ષમ so you can give them one more word, for e.g.
અક્ષમ/નિષ્ક્રિય which will help them understand your instructions easily.